src/test/test-seccomp.c

```diff
@@ -216,6 +218,7 @@ static void test_restrict_namespace(void) {
         }
 
         assert_se(wait_for_terminate_and_warn("nsseccomp", pid, true) == EXIT_SUCCESS);
+#endif
```

Comment:
see comment about that in git msg...

i see no reason why we couldn't support all archs here. the only reason why an arch isn't supported is that we don't know the right order, not that it wouldn't work. 

Because of that I figured this is different from the other cases, and not testing this for now is OK, under the assumption that porters will see the warning send patches and we'll start testing the full set sooner or later...